application-tour-ui/src/main/resources/TourCode/TourJson.xml

```diff
@@ -110,13 +119,19 @@
           #if ($element == 'body')
             #set ($element = '')
           #end
+          #if ($element == '')
+            #set ($orphan = true)
+          #else
+            #set ($orphan = false)
+          #end
```

Comment:
I think I had tried something like the first suggestion but Velocity was throwing an error at me, I must've been doing something wrong. I'll commit something to change it.